src/main/java/frc/pathrecognizer/PixyCameraConnector.java

```diff
@@ -52,6 +54,29 @@ public PixyCameraConnector() {
             block.getAngle()
           );
 
+          switch  (count){
+          case 1:
+          {
+          logger.log("Pixel Width 1:", block.getWidth());
+          logger.log("PixelHeight 1:", block.getHeight());
+          break;
+          }
+          case 2:
+          {
+          logger.log("Pixel Width 2: ", block.getWidth());
+          logger.log("PixelHeight: 2: ", block.getHeight());
+          break;
+          }
+          case 3:
+          {
+          logger.log("Pixel Width 3: ", block.getWidth());
+          logger.log("PixelHeight 3: ", block.getHeight());
```

Comment:
at this point, alarm bells MUST have been going off! the same code 3 times? please NO